### PR TITLE
changed header buttons order

### PR DESCRIPTION
**Pull request by maxMarta**
Changed header buttons order. Added "buy ticket" to header buttons

### CODE REVIEW

**Comment by maxMarta** [comment]
**Desktop**
http://my.jetscreenshot.com/demo/20180109-6vnk-29kb.jpg

**Mobile**
http://my.jetscreenshot.com/demo/20180109-u2pn-32kb.jpg

**Comment by ozasadnyy** [comment]
Let's try a design for drawer like this one:

File: src/hoverboard-app.html [hunk 1]
```diff
@@ -121,8 +124,15 @@
         }
       }
 
-      .buy-button {
-        margin-left: 12px;
+      .buy-ticket{
```
Comment (sophieH29) [inline src/hoverboard-app.html]: space after .buy-ticket :)

File: src/hoverboard-app.html [hunk 2]
```diff
@@ -71,11 +71,14 @@
       }
 
       .drawer-list a {
-        padding: 8px 24px;
         display: block;
         color: var(--primary-text-color);
       }
 
+      app-drawer a {
```
Comment (sophieH29) [inline src/hoverboard-app.html]: should it apply to all <a> elements, or only .buy-ticket?

**Comment by maxMarta** [comment]
Changed button in drawer as discussed
jgx4yofcwya

File: src/hoverboard-app.html (hunk 2 above)
Comment (maxMarta) [reply src/hoverboard-app.html]: It's applied to .drawer-list a, but the same padding should be used to .buy-ticket

**Comment by ozasadnyy** [comment]
Looks good, one last thing. Padding on the top and bottom of the menu should be the same
image

**Comment by maxMarta** [comment]
Fixed
http://my.jetscreenshot.com/demo/20180117-byod-39kb.jpg

**Pull request merged**